cobc/pplex.l

```diff
@@ -1867,6 +1883,14 @@ int cobc_deciph_source_format (const char *sfname) {
 	return 1;
 }
 
+int cobc_deciph_header_filename (const char *filename) {
+	size_t n = strlen(filename);
+	if (filename[n-2] == '.' && filename[n-1] == 'h') {
```

Comment:
I think this breaks support for `>> IMP INCLUDE "<system.h>`. In any case it should be handled identical as `--include` does.